### PR TITLE
Futureproof sitenow events code

### PR DESCRIPTION
**Pull request by joewhitsitt**
Please don't take my word for it and test. I am stumbling around with this kind of coding...

### CODE REVIEW

File: docroot/modules/custom/sitenow_events/src/Controller/EventsController.php [hunk 1]
```diff
@@ -2,14 +2,42 @@
...
 
 /**
  * Returns responses for University of Iowa Events Single routes.
  */
 class EventsController extends ControllerBase {
 
+  /**
+   * Config.
+   *
+   * @var \Drupal\Core\Config\ImmutableConfig
```
Comment (richardbporter) [inline docroot/modules/custom/sitenow_events/src/Controller/EventsController.php]: This should match the parameter type in the constructor docblock.

**Review by richardbporter — CHANGES_REQUESTED** [review]

**Comment by richardbporter** [comment]
Do you see the link to the configure form in Admin -> Configuration locally? Could be my environment but I'm not.

**Review by richardbporter — APPROVED** [review]

**Comment by richardbporter** [comment]
I fixed the menu link issue. Are webmasters supposed to have permission to manage sitenow events?

**Comment by joewhitsitt** [comment]
yes

**Review by richardbporter — APPROVED** [review]

**Pull request merged**